schemas/README.md

```diff
@@ -1593,6 +2172,548 @@ Raw image data
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>timestamp</code></td>
+<td>
+
+time
+
+</td>
+<td>
+
+Timestamp of the entity
+
+</td>
+</tr>
+<tr>
+<td><code>frame_id</code></td>
+<td>
+
+string
+
+</td>
+<td>
+
+Frame of reference
+
+</td>
+</tr>
+<tr>
+<td><code>id</code></td>
+<td>
+
+string
+
+</td>
+<td>
+
+Identifier for the entity. A entity will replace any prior entity on the same topic with the same `id`.
+
+</td>
+</tr>
+<tr>
+<td><code>lifetime</code></td>
+<td>
+
+duration
+
+</td>
+<td>
+
+Length of time (relative to `timestamp`) after which the entity should be automatically removed. Zero value indicates the entity should remain visible until it is replaced or deleted.
+
+</td>
+</tr>
+<tr>
+<td><code>frame_locked</code></td>
+<td>
+
+boolean
+
+</td>
+<td>
+
+Whether the entity should keep its location in the fixed frame (false) or follow the frame specified in `frame_id` as it moves relative to the fixed frame (true)
+
+</td>
+</tr>
+<tr>
+<td><code>metadata</code></td>
+<td>
+
+[KeyValuePair](#keyvaluepair)[]
+
+</td>
+<td>
+
+Additional user-provided metadata associated with the entity. Keys must be unique.
+
+</td>
+</tr>
+<tr>
+<td><code>arrows</code></td>
+<td>
+
+[ArrowPrimitive](#arrowprimitive)[]
+
+</td>
+<td>
+
+Arrow primitives
+
+</td>
+</tr>
+<tr>
+<td><code>cubes</code></td>
+<td>
+
+[CubePrimitive](#cubeprimitive)[]
+
+</td>
+<td>
+
+Cube primitives
+
+</td>
+</tr>
+<tr>
+<td><code>spheres</code></td>
+<td>
+
+[SpherePrimitive](#sphereprimitive)[]
+
+</td>
+<td>
+
+Sphere primitives
+
+</td>
+</tr>
+<tr>
+<td><code>cylinders</code></td>
+<td>
+
+[CylinderPrimitive](#cylinderprimitive)[]
+
+</td>
+<td>
+
+Cylinder primitives
+
+</td>
+</tr>
+<tr>
+<td><code>lines</code></td>
+<td>
+
+[LinePrimitive](#lineprimitive)[]
+
+</td>
+<td>
+
+Line primitives
+
+</td>
+</tr>
+<tr>
+<td><code>triangles</code></td>
+<td>
+
+[TriangleListPrimitive](#trianglelistprimitive)[]
+
+</td>
+<td>
+
+Triangle list primitives
+
+</td>
+</tr>
+<tr>
+<td><code>texts</code></td>
+<td>
+
+[TextPrimitive](#textprimitive)[]
+
+</td>
+<td>
+
+Text primitives
+
+</td>
+</tr>
+<tr>
+<td><code>models</code></td>
+<td>
+
+[ModelPrimitive](#modelprimitive)[]
+
+</td>
+<td>
+
+Model primitives
+
+</td>
+</tr>
+</table>
+
+## SceneEntityDeletion
+
+Command to remove previously published entities
+
+<table>
+  <tr>
+    <th>field</th>
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>timestamp</code></td>
+<td>
+
+time
+
+</td>
+<td>
+
+Timestamp of the deletion. Only matching entities earlier than this timestamp will be deleted.
+
+</td>
+</tr>
+<tr>
+<td><code>type</code></td>
+<td>
+
+[enum SceneEntityDeletionType](#enum-sceneentitydeletiontype)
+
+</td>
+<td>
+
+Type of deletion action to perform
+
+</td>
+</tr>
+<tr>
+<td><code>id</code></td>
+<td>
+
+string
+
+</td>
+<td>
+
+Numeric identifier which must match if `kind` is `MATCHING_ID`.
+
+</td>
+</tr>
+</table>
+
+## SceneEntityUpdate
```

Comment:
It feels like this schema should be called `SceneEntities`, in keeping with all our other array type schemas. 

I know it technically contains SceneEntityDeletion as well, but that is still a type of SceneEntity.

Reply:
Or what about SceneEntityUpdates?

Reply:
Another idea: SceneUpdate(s)?

Reply:
I like `SceneUpdate`